pkgs/development/python-modules/flask-gravatar/default.nix

```diff
@@ -0,0 +1,62 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, flask
+, pytestCheckHook
+, pycodestyle
+, pytest-cov
+, pygments
+}:
+
+buildPythonPackage rec {
+  pname = "flask-gravatar";
+  version = "0.5.0";
+
+  src = fetchPypi {
+    pname = "Flask-Gravatar";
+    inherit version;
+    sha256 = "YGZfMcLGEokdto/4Aek+06CIHGyOw0arxk0qmSP1YuE=";
+  };
+
+
+  preBuild = ''
+    sed -i setup.py \
+     -e "s|tests_require=tests_require,||g" \
+     -e "s|extras_require=extras_require,||g" \
+     -e "s|setup_requires=setup_requires,||g"
+  '';
+
+  propagatedBuildInputs = [
+    flask
+  ];
+
+  # pep8 is deprecated
+  preCheck = ''
+    substituteInPlace pytest.ini --replace "--pep8" ""
+  '';
+
+  checkInputs = [
+    pytestCheckHook
+    pycodestyle
+    pytest-cov
+    pygments
+  ];
+
+  # use pycodestyle as a replacement for the removed pep8.
+  # Since there is no pytest-pycodestyle plugin in nixpkgs
+  # use it separately after the pytestCheckHook
+  postCheck = ''
+    ${pycodestyle}/bin/pycodestyle --statistics --ignore=E305,E501,W293 \
+                                   --exclude=docs/conf.py,docs/_themes/flask_theme_support.py
```

Comment:
Does this only check for style things or has real tests? if it is only the former we can skip it.

Reply:
the original tests do unittests and also a pep8 check. I replaced the pep8 check with the newer pycodestyle. 
Sure, I''ll remove it